src/scss/components/_form.scss

```diff
@@ -7,6 +7,34 @@
 **/
 
 .form {
+  position: relative;
+  overflow: hidden;
+
+  &__envelop {
+    background: get-color(reward);
+    width: 100%;
+    position: relative;
+    padding: 30px 30px 20px 30px;
+
+    &__title {
```

Comment:
@iondrimba on BEM methodology we don't have a element inside of another element. My suggestion here is move this element to the same level of `form__element`.